addons/mail/static/src/models/attachment_viewer/attachment_viewer.js

```diff
@@ -20,6 +22,71 @@ function factory(dependencies) {
                 dialog.delete();
             }
         }
+
+        /**
+         * Handle onclick on next button
+         *
+         * @param {MouseEvent} ev
+         */
+        onClickNext(ev) {
+            markEventHandled(ev, 'attachmentViewer.clickNext');
+            this._next();
+        }
+
+        /**
+         * Handle onclick on previous button
+         *
+         * @param {MouseEvent} ev
+         */
+        onClickPrevious(ev) {
+            markEventHandled(ev, 'attachmentViewer.clickPrevious');
+            this._previous();
+        }
+
+        /**
+         * Handle keydown even inside the attachment viewer.
+         *
+         * @param {MouseEvent} ev
+         * @param {String} direction - arrow direction (possible value: next, previous).
+         */
+        onKeydown(ev, direction) {
```

Comment:
the key that is pressed should be read from ev itself, there is no need for an extra param here